### PR TITLE
Update Spanish.properties

### PR DESCRIPTION
**Pull request by ArchDuque-Pancake**

### CODE REVIEW

File: android/assets/jsons/translations/Spanish.properties [hunk 1]
```diff
@@ -86,12 +86,9 @@ We will remember this. = ¡Recordaremos esto!
 [civName] and [targetCivName] have signed a Peace Treaty! = ¡[civName] y [targetCivName] han firmado un tratado de paz!
 [civName] and [targetCivName] have signed the Declaration of Friendship! = ¡[civName] y [targetCivName] han firmado una declaración de amistad!
 [civName] has denounced [targetCivName]! = ¡[civName] ha denunciado a [targetCivName]!
- # Requires translation!
-Do you want to break your promise to [leaderName]? = 
- # Requires translation!
-We promised not to settle near them ([count] turns remaining) = 
- # Requires translation!
-They promised not to settle near us ([count] turns remaining) = 
+Do you want to break your promise to c? = ¿Quieres romper tú promesa con [leaderName]?
```
Comment (xlenstra) [inline android/assets/jsons/translations/Spanish.properties:89]: I believe you misplaced a tag here

Comment (ArchDuque-Pancake) [reply android/assets/jsons/translations/Spanish.properties:89]: !!!

**Comment by ArchDuque-Pancake** [comment]
Fixxed it

**Comment by yairm210** [comment]
merge translations

**Comment by yairm210** [comment]
Translations branch created

**Comment by yairm210** [comment]
Translations PR created

**Pull request merged**